.gitattributes

```diff
@@ -1 +1 @@
-* text=auto
+* text eol=lf
```

Comment:
This wasn't part of 552bf993592c42f99ec0f3f6afcf4ef1906c51a6, but I went ahead and included it as part of the merge.